data/pathfinder/paizo/roleplaying_game/advanced_race_guide/arg_feats.lst

```diff
@@ -107,9 +107,9 @@ Courageous Resolve			CATEGORY:FEAT		TYPE:General	PREABILITY:2,CATEGORY=Special A
-Lucky Healer				CATEGORY:FEAT		TYPE:General	PREABILITY:1,CATEGORY=Special Ability,Halfling ~ Adaptable Luck																															PRERACE:1,Halfling,RACESUBTYPE=Halfling																																																	DESC:Your luck allows you to draw from magical healing far more efficiently than most.																																																																																																																																																																																																																																																																																																																																																																																																																																																																																				SOURCEPAGE:p.66											BENEFIT:When a magical healing effect (such as a spell with �cure� in the title or channel energy) cures you, you can spend one use of your adaptable luck racial trait to reroll the amount of damage cured. You regain a number of hit points equal to the new roll or the original roll, whichever is greater. Other creatures healed by the effect do not gain this benefit.
+Lucky Healer				CATEGORY:FEAT		TYPE:General	PREABILITY:1,CATEGORY=Special Ability,Halfling ~ Adaptable Luck																															PRERACE:1,Halfling,RACESUBTYPE=Halfling																																																	DESC:Your luck allows you to draw from magical healing far more efficiently than most.																																																																																																																																																																																																																																																																																																																																																																																																																																																																																				SOURCEPAGE:p.66											BENEFIT:When a magical healing effect (such as a spell with “cure” in the title or channel energy) cures you, you can spend one use of your adaptable luck racial trait to reroll the amount of damage cured. You regain a number of hit points equal to the new roll or the original roll, whichever is greater. Other creatures healed by the effect do not gain this benefit.
```

Comment:
Didn't change anything to _Lucky Healer_ and _Risky Striker_, just some non significant whitespace I guess.